src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/components/MethodologyStatusForm.tsx

```diff
@@ -26,6 +27,7 @@ export interface FormValues {
 interface Props {
   isPublished?: string;
   methodology: MethodologyVersion;
+  statusPermissions: MethodologyStatusPermissions | undefined;
```

Comment:
```suggestion
  statusPermissions?: MethodologyStatusPermissions;
```

Reply:
I did a quick skim over the code base for similar instances and changed them too. You might want to check over them.